vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt

```diff
@@ -171,14 +189,21 @@ class CreateDirectRoomActivity : SimpleFragmentActivity() {
         }
     }
 
-    private fun renderCreationSuccess(roomId: String?) {
+    private fun renderCreationSuccess(roomId: String) {
```

Comment:
It will be hard to report every little fixes like that in Element but ideally this is what you should do (or fix only on Element and get it back during the next merge).

Reply:
Done: https://github.com/vector-im/element-android/pull/4757/commits/c13feb983ec6b05437aa5cb76fc3a7c35cad52ba

Reply:
Thanks!